resources/assets/less/bem/not-official-footer.less

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.not-official-footer {
+  width: 100%;
+  padding: 10px;
+  background: @red;
+  text-align: center;
+
+  @media @mobile {
+    font-size: @font-size--title-small;
+  }
+
+  &__link {
+    .link-white();
+    text-decoration: underline;
```

Comment:
Maybe use `--hsl-c2` color instead. I don't think there's anywhere else where link is underlined

Reply:
![image](https://user-images.githubusercontent.com/4964985/135969223-2422f04c-f241-40aa-8aa1-5d0dc2f834c3.png)

This is the combination of `--hsl-red-3` background colour and `--hsl-c2` link colour. Is it okay?

Reply:
yes